packages/website/docs/creating-a-renderer.md

```diff
@@ -274,7 +274,7 @@ function Autocomplete() {
 
   React.useEffect(() => {
     if (!(formRef.current && panelRef.current && inputRef.current)) {
-      return;
+      return = true;
```

Comment:
```suggestion
      return undefined;
```

This would make more sense, although ideally we should ignore this ESLint rule in this context.